src/averageAges.js

```diff
@@ -36,8 +38,15 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
+function calculateWomenAverageAge(people, withChildren = false) {
   // write code here
+  const females = people.filter(female =>
```

Comment:
Why is it `female` here, if it's just a person, you don't know what gender it is yet

Reply:
In this task, it is stated that we have the average age of women with and without children, so the name is chosen accordingly

* Implement calculateWomenAverageAge function
 *
 * Function returns average age of women in array. If `withChildren` is
 * specified then function calculates average age only for women with children
 *
 * Hint: To check if a woman has children you should find someone who mention
 * her as mother.